package.json

```diff
@@ -35,6 +38,7 @@
     "select2": "~3.5.2-browserify"
   },
   "scripts": {
-    "lint": "eslint -c ./.eslintrc webpack/assets/**/*.js  || exit 0"
+    "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/assets/**/*.js",
```

Comment:
The `|| exit 0` is there because if eslint finds errors it exits with a non-zero status, causing npm to think there was a bug in eslint and leading to confusing error message being displayed.